packages/dd-trace/src/runtime_metrics/index.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+let runtimeMetrics
+
+const noop = runtimeMetrics = {
+  start: () => {},
+  stop: () => {},
+  track: () => {},
+  boolean: () => {},
+  histogram: () => {},
+  count: () => {},
+  gauge: () => {},
+  increment: () => {},
+  decrement: () => {}
```

Comment:
Super not important nit: any reason not to use

```suggestion
  decrement() {}
```

If we talk about super optimizing startup time, this will safe byes loaded and that need to be read :D